Makefile

```diff
@@ -425,9 +425,15 @@ cover: ## Runs the golang coverage tool. You must run the unit tests first.
 	$(GO) tool cover -html=cover.out
 	$(GO) tool cover -html=ecover.out
 
-test-data: start-docker ## Add test data to the local instance.
-	$(GO) run $(GOFLAGS) -ldflags '$(LDFLAGS)' $(PLATFORM_FILES) config set TeamSettings.MaxUsersPerTeam 100
-	$(GO) run $(GOFLAGS) -ldflags '$(LDFLAGS)' $(PLATFORM_FILES) sampledata -w 4 -u 60
+test-data: run-server ## Add test data to the local instance.
+	@if ! ./scripts/wait-for-system-start.sh; then \
+		make stop; \
+	fi
+
+	@echo ServiceSettings.EnableLocalMode must be set to true.
```

Comment:
Wondering if we should automate this using `jq` on `config.json` and force a server restart instead to make sure it gets applied. After all, this is a dev environment we are talking about.

Reply:
I was debating that and thought it seemed like a bit of an overreach and was preferring to iterate on that addition if we think it's necessary in the future. Thoughts?

Reply:
Fair enough :+1: